crates/data_models/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "data_models"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Please include more of the manifest fields, can refer to any of the other crates.